content/logo/index.mdx

```diff
@@ -106,7 +106,7 @@ _body:
             linkContent: >
               <UtilityButton buttonText="LOGO OPTIONS" size="small"
               btnIcon="BsArrowRightCircle" removeTopMargin={true}
-              link="https://www.ssw.com.au/ssw/logo/NETUG" />
+              link="/logo/net-ug" />
```

Comment:
maybe we should use `/logo/netug` to make it consistent with other routes (i.e /netug/brisbane)

Reply:
Agree. I have made the change to `/logo/netug`